openlibrary/templates/type/edition/view.html

```diff
@@ -79,7 +79,7 @@
         schema_attribute = cond(include_schema, ' itemprop="author"', '')
         authors = cond(work, work and work.get_authors(), edition and edition.get_authors())
         if authors:
-            return ', '.join('<a href="%s"%s>%s</a>' % (a.url(), schema_attribute, truncate(a.name, 40)) for a in authors)
+            return ', '.join('<a href="%s"%s>%s</a>' % (a.url(), schema_attribute, truncate(a.name, 40)) if a.name else "<em>Unknown author</em>" for a in authors)
```

Comment:
```suggestion
            return ', '.join('<a href="%s"%s>%s</a>' % (a.url(), schema_attribute, truncate(a.name or "<em>Unknown author</em>", 40)) for a in authors)
```

Reply:
A side effect of this suggestion would be linking to the corrupt author entity.  I submitted [another PR for a related issue](https://github.com/internetarchive/openlibrary/pull/3996) where I also intentionally didn't link to the corrupt author entity, thinking that interested people could get it from json data.  It seems like we should be consistent.  Is your preference to link to corrupted author entities?

Reply:
Thanks.  Now I understand the intention.  I modified the suggestion above.